### PR TITLE
Git+ssh url support

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Support url schema `git+ssh` and optional `-e` for `git+ssh` and `git+git` urls.

cf https://pip.pypa.io/en/stable/reference/pip_install/#git

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
The README would need to be updated. And probably a minor release to be made.
:)

File: tests/unit/test_install.py [hunk 1]
```diff
@@ -73,12 +73,30 @@ def test_strips_editable_flag_from_vcs_urls(self):
...
         self.assertEqual(ret, ['git+https://github.com/ByteInternet/...'])
 
-    def test_transforms_vcs_ssh_url_to_oauth(self):
+    def test_transforms_vcs_git_url_to_oauth(self):
+        fname = self._create_reqs_file(['git+git@github.com:ByteInternet/...'])
+
+        ret = collect_requirements(fname, transform_with_token='my-token')
+        self.assertEqual(ret, ['git+https://my-token:x-oauth-basic@github.com/ByteInternet/...'])
+
+    def test_transforms_vcs_git_url_to_oauth_dashe_option(self):
```
Comment (vdloo) [inline tests/unit/test_install.py]: dashe?

**Comment by vdloo** [comment]
> The README would need to be updated.

Could you add that to this branch?

**Comment by vdloo** [comment]
Otherwise :+1:

File: tests/unit/test_install.py (hunk 1 above)
Comment (AdrieanKhisbe) [reply tests/unit/test_install.py]: `-e`

Comment (AdrieanKhisbe) [reply tests/unit/test_install.py]: 😉

**Comment by AdrieanKhisbe** [comment]
Just updated readme, tell me if it's good for you 😃

**Comment by AdrieanKhisbe** [comment]
great! 😃

**Pull request merged**